lib/rubocop/cop/style/endless_method.rb

```diff
@@ -5,8 +5,11 @@ module Cop
     module Style
       # Checks for endless methods.
       #
-      # It can enforce either the use of endless methods definitions
-      # for single-lined method bodies, or disallow endless methods.
+      # It can enforce either the use of endless methods definitions for single-lined method bodies,
+      # or disallow endless methods.
+      #
+      # `require_always` style enforces endless method definitions for a single statement
+      # (not a single line) containing a method chain with a line break.
```

Comment:
I think the second line of this paragraph is a bit confusing, because the implementation of this style doesn't check whether there are any line breaks, and doesn't care if it's a method chain och just a numeric literal for example. So:

`require_always` style enforces endless method definitions for single statement methods.

Reply:
I like the proposed text, it's simple! 🙏  I've update it!